src/components/Footer/Footer.scss

```
@@ -0,0 +1,94 @@
+.footer {
+  height: 100vh;
+  position: relative;
+  color: #d6d5cb;
+
+  .container {
+    width: 100vw;
+    padding-bottom: 1em;
+    padding-right: 2em;
+    display: flex;
+    justify-content: space-between;
+    background-color: #333;
+
+    .item {
+      margin-left: 1em;
+      width: 24em;
+      padding: 0 1em;
+      .bottomBorderBox {
+        border-bottom: 1px solid #d6d5cb;
+        padding: 1.5em 0;
+        width: 97%;
+      }
```

```suggestion
    .item {
      margin-left: 1em;
      width: 24em;
      padding: 0 1em;
      
      .bottomBorderBox {
        border-bottom: 1px solid #d6d5cb;
        padding: 1.5em 0;
        width: 97%;
      }
```
위코드의 css코드 작성 컨벤션 입니다. 컨벤션에 맞게 수정해주세요
1. selector 윗줄은 공백을 줘서 가독성을 높인다.
2. 속성과 속성사이 의미없는 공백은 삭제한다.
요 두가지 컨벤션 앞으로도 잘 지켜서 작성해주세요